### ISSUE TITLE
Problem with Auto-Mount of USB drive in Caja

### ISSUE THREAD

**Issue by dooleysworne** [mate-desktop/caja#1328]
#### Expected behaviour
Auto mount of USB drive in Caja should be seamless and display the contents when opened.

#### Actual behaviour
Device is displayed in side panel, but when clicked, an error pops up showing "Unable to mount X.X GB Volume" "Not authorized to perform operation". See image
![image](https://user-images.githubusercontent.com/35536000/70916553-7b99c880-1fe1-11ea-8d09-d2f8bbf3151f.png)

#### Steps to reproduce the behaviour
Happens with all USB drives regardless of formatting (ntfs, ext4, fat32).  USB drive can be mounted using command line and mount point (with sudo). Once mounted via CLI, it becomes accessible in Caja.

#### MATE general version
MATE 1.22.2

#### Package version
Caja 1.22.2

#### Linux Distribution
Ubuntu 19.10 (Eoan Ermine)

**Comment by lukefromdc** [comment]
When you mount a drive from the command line you are using sudo and would require sudo priviliges to unmount the drive as well as it was effectively mounted by root. If you run "sudo caja" you should then be able to unmount the drive from it, thus confirming the permissions issue.

Do you have mate-polkit installed? If so you should get a dialog asking for the sudo passphrase to unmount the volume. If it is not installed or not running (e.g. outside a MATE session), you would get this dialog

**Comment by dooleysworne** [comment]
You are correct that it requires sudo privileges to unmount the drive. When I run "sudo caja", it shows the same error, but does not display the drive in the side panel. I installed mate-polkit and got this response from the system: "Unable to register authentication agent . . . PolicyKit1.Error.Failed: An authentication agent already exists for the given subject".  I'm confounded.

**Comment by lukefromdc** [comment]
If you are running caja in a.session that has its own polkit manager
such as GNOME or Cinnamon this is expected. When mounting from 
the command line you may simply have to unmount from the command
line if not using your session's native file manager.

**Comment by dooleysworne** [comment]
Just discovered that my machine will not mount CDs or DVDs either. I decided to reboot the laptop with a live Ubuntu MATE 18.04 LTS DVD to see if it was a machine issue. After booting to the live OS, I inserted a USB stick in the port and caja mounted the USB drive just as it should.

**Comment by raveit65** [comment]
Congratulation, you only need to install Ubuntu-MATE to fix the issue :)

**Comment by dooleysworne** [comment]
Currently running Ubuntu MATE 19.10. That's what has me buffaloed.
